src/cpp/desktop/DesktopWebView.cpp

```diff
@@ -229,10 +271,39 @@ void WebView::contextMenuEvent(QContextMenuEvent* event)
       menu->addAction(webPage()->action(QWebEnginePage::SelectAll));
    }
    
-#ifndef NDEBUG
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
+   
+   menu->addSeparator();
+   menu->addAction(tr("&Reload"), [&]() { triggerPageAction(QWebEnginePage::Reload); });
```

Comment:
Does Reload only work on 5.11?

Reply:
It works on both; we have a separate branch below that handles adding the Reload + Inspect Element items for older Qt.